assets/app/view/game/track_conversion.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'lib/settings'
+require 'view/game/actionable'
+
+module View
+  module Game
+    class TrackConversion < Snabberb::Component
+      include Actionable
+      include Lib::Settings
+
+      needs :routes, store: true, default: []
+      needs :selected_route, store: true, default: nil
+
+      # Get routes that have a length greater than zero
+      # Due to the way this and the map hook up routes needs to have
+      # an entry, but that route is not valid at zero length
+      def active_routes
+        @routes.select { |r| r.chains.any? }
+      end
+
+      def render
+        @step = @game.active_step
+        current_entity = @game.round.current_entity
+
+        trains = @game.conversion_trains(current_entity)
+
+        train_help =
+          if (helps = @game.train_help(current_entity, trains, @routes)).any?
+            h('ul',
+              { style: { 'padding-left': '20px' } },
+              helps.map { |help| h('li', [h('p.small_font', help)]) })
+          end
+
+        if !@selected_route && (first_train = trains[0])
+          route = Engine::Route.new(@game, @game.phase, first_train, abilities: @abilities, routes: @routes)
+          @routes << route
+          store(:routes, @routes, skip: true)
+          store(:selected_route, route, skip: true)
+        end
+
+        @routes.each(&:clear_cache!)
+
+        trains = trains.flat_map do |train|
+          onclick = lambda do
+            unless (route = @routes.find { |t| t.train == train })
+              route = Engine::Route.new(@game, @game.phase, train, abilities: @abilities, routes: @routes)
+              @routes << route
+              store(:routes, @routes)
```

Comment:
skip

Reply:
I'm afraid I don't understand. You want a skip in the lambda?

Reply:
No, I think you want the page to be redrawn when you select a new route (this is the lamba associated with clicking on a train - the highlighting needs to change from one route to another).

Reply:
but you already call store(:selected_route) right under it?

Reply:
OK. Got it.